src/format/parsed.rs

```diff
@@ -580,7 +580,7 @@ impl Parsed {
             (None, Some(_)) => 0, // UNIX timestamp may assume 0 offset
             (None, None) => return Err(NOT_ENOUGH),
         };
-        let datetime = self.to_naive_datetime_with_offset(offset)?;
+        let datetime = self.to_naive_datetime_with_offset_inner(offset, false)?;
```

Comment:
Why not?

Reply:
If a date includes an offset in hours and maybe minutes, I think we should require the date and time to also have a precision of hours and minutes without fallback.